src/hooks/useDatasetManager.js

```diff
@@ -136,18 +131,26 @@ export const useDatasetManager = () => {
     return response
   }
 
+  // checks if the given dataset ID is My dataset ID
+  const isMyDatasetId = (id) => id === datasetId
+
   // takes download options, and optional dataset ID and one-off experiment accession code
   const startProcessingDataset = async (
```

Comment:
We will create 2 new issues for cleaning up the communication between formik and useDatasetManager.